Fix Editions (and some refactoring)

SiAdcock opened the pull request:
Fixes an issue we introduced in #9 whereby links pointed to the Android app repo instead of the Editions repo.

I've also done a bit of refactoring to make the code clearer, and improved the developer experience slightly by:

- ignoring `node_modules`
- upgrading Node to v10
- adding an npm script that kicks of the `builds.gutools` dev server
- updating the README

SiAdcock committed:
ignore things

SiAdcock committed:
allow devs to run  builds

SiAdcock committed:
add docs for devs

SiAdcock committed:
use latest node

SiAdcock committed:
fix reference bug

SiAdcock committed:
refactor initial data

SiAdcock committed:
replace jquery with fetch

SiAdcock commented on builds/index.html:
```diff
@@ -60,29 +60,59 @@ <h1 class="text-center">Daily Edition</h1>
-              <a :href="releases.android.development.teamcity">{{ releases.androidEditions.development.date }}</a> 
-              <a :href="releases.android.development.github">#{{ releases.androidEditions.development.sha1 }}</a></p>
+              <a :href="releases.androidEditions.development.teamcity">{{ releases.androidEditions.development.date }}</a> 
+              <a :href="releases.androidEditions.development.github">#{{ releases.androidEditions.development.sha1 }}</a></p>
```
The bug fix!

SiAdcock committed:
remove console.log

jacobwinch reviewed (approved):
Looks great 👍

jorgeazevedo commented:
Do you know if `reduce`, `fetch` and arrow functions are available on Android 5 and iOS 9 devices? I'm thinking at least arrow functions aren't available in iOS 9, according to #6

SiAdcock committed:
make safari 9 compatible

SiAdcock committed:
add browser support note

SiAdcock commented:
@jorgeazevedo really good points!

- reverted arrow functions and fancy computed property syntax
- included a `fetch` polyfill as I feel icky about using jQuery just for AJAX sugar
- `reduce` is part of ES5 and [is supported by iOS Safari 9](https://caniuse.com/#feat=es5)
- added a note about browser support to the README

jorgeazevedo commented:
Thanks for adding that note to the README. It's not written anywhere that this needs to support the same iOS/Android versions that our apps support, sorry about that!

SiAdcock committed:
Better readme note
